module/Motion/Rotation.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file   Rotation.h
+ * @brief  回頭動作の中間クラス
+ * @author desty505
+ */
+
+#ifndef ROTATION_H
+#define ROTATION_H
+
+#include "Motion.h"
+#include "Mileage.h"
+#include "SpeedCalculator.h"
+#include "SystemInfo.h"
+
+class Rotation : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   * @param _targetValue 回転角度(deg) 0~360
+   * @param _targetSpeed 目標速度
+   * @param _isClockwise 回頭方向 ture:時計回り, false:反時計回り
+   */
+  Rotation(double _targetSpeed, bool _isClockwise);
+
+  /**
+   * @brief 回頭する
+   */
+  void run();
+
+  /**
+   * @brief 直進する際の事前条件判定をする
+   * @param targetSpeed 目標速度
+   * @note オーバーライド必須
+   */
+  virtual bool runPreconditionJudgement(double targetSpeed);
```

Comment:
```suggestion
  virtual bool isMetPrecondition(double targetSpeed);
```

とかで
true: 満たしている
false: 満たしていない
の方が直感的だと思った(任意)